### PR TITLE
Update English_admin_featurebox.php

### PR DESCRIPTION
**Pull request by yesszus**
Add phrases.

### CODE REVIEW

File: e107_plugins/featurebox/admin_config.php [hunk 1]
```diff
@@ -38,10 +38,10 @@ class fb_admin extends e_admin_dispatcher
 	);	
 
 	protected $adminMenu = array(
-		'main/list'			=> array('caption'=> 'Featurebox List', 'perm' => 'P'),
+		'main/list'			=> array('caption'=> 'Featurebox&nbsp;'.LAN_LIST', 'perm' => 'P'),
 		'main/create'		=> array('caption'=> LAN_CREATE, 'perm' => 'P'),
 		'category/list' 	=> array('caption'=> LAN_CATEGORIES, 'perm' => 'P'),
-		'category/create'	=> array('caption'=> "Create Category", 'perm' => 'P'),
+		'category/create'	=> array('caption'=> FBLAN_25, 'perm' => 'P'),
```
Comment (CaMer0n) [inline e107_plugins/featurebox/admin_config.php]: Why not use the generic `LAN_CREATE_CATEGORY` ?

**Comment by yesszus** [comment]
@CaMer0n updated all.

**Comment by CaMer0n** [comment]
👍

**Comment by CaMer0n** [comment]
btw. These changes broke the admin area with a 'fatal error'.

**Pull request merged**